scylla/src/transport/load_balancing/token_and_latency_aware.rs

```diff
@@ -0,0 +1,305 @@
+use super::{
+    ChildLoadBalancingPolicy, LatencyAwarePolicy, LoadBalancingPolicy, Plan, Statement,
+    TokenAwarePolicy,
+};
+use crate::transport::cluster::ClusterData;
+use std::time::Duration;
+
+/// A token- and latency-aware load balancing policy.
+#[derive(Debug)]
+pub struct TokenAndLatencyAwarePolicy {
```

Comment:
I guess this should be removed now? I think it is possible now to compose `LatencyAwarePolicy` with `TokenAwarePolicy` manually.

Reply:
The tests might be useful though in order to verify that the policies compose in a sane way.